python/tskit/trees.py

```diff
@@ -2421,8 +2448,7 @@ def individuals(self):
         :return: An iterator over all individuals.
         :rtype: iter(:class:`.Individual`)
         """
-        for j in range(self.num_individuals):
-            yield self.individual(j)
+        return _ReversibleIterator(self.individual, self.num_individuals)
```

Comment:
This changes the semantics quite a lot though. Now, you could write 
```
ts.individuals()[0] 
```
It's not an iterator any more, it's a Sequence. Not that this is necessarily bad, it's just **different** and we need to spend time thinking through the consequences when we change the semantics of things.